### DISCUSSION TITLE
Include sub-folders of any folder defined in the configuration

### DISCUSSION THREAD

**Discussion by ScottL7** [davish/obsidian-full-calendar#458] [Feature Requests]
### ⚠️ Please check that this feature request hasn't been suggested before.

- [X] I searched previous [Ideas in Discussions](https://github.com/davish/obsidian-full-calendar/discussions/categories/feature-requests) didn't find any similar feature requests.
- [X] I searched previous [Issues](https://github.com/davish/obsidian-full-calendar/issues) didn't find any similar feature requests.

### 🔖 Feature description

I automatically break down my daily journals into a folder structure similar to the following: Journal/Daily Notes/YYYY-MM. Unfortunately, I must add a new folder for each month manually.

### ✔️ Solution

Include all files in sub-folders, recursively.

**Comment by Spiralo-Idioide** [comment]
Definitely agree that this feature should be part of the plugin !
For the moment I am not updating to most recents versions to keep this feature. And it is a bad idea in the long term.

**Comment by eduardoRod** [comment]
Exactly. Came here just to request this feature. 

In my case I sometimes group meetings by project. 

A suggestion would be to add a calendar based on folder path OR tags

**Comment by UjCbFwtBayFM** [comment]
I support this feature suggestion.
I'm dealing with a lot of files in my calendar and I'd love to be able to use a Calendar/Year/Month folder structure.

**Comment by OdinsHat** [comment]
If this was created it would also add support for the [Obsidian Journal plugin](https://github.com/srg-kostyrko/obsidian-journal) which is a successor to Periodic Notes - which is abandoned.

Obsidian Journal uses this sort of folder structure if you set it up that way.

Here's a visual representation of the folder structure:
![image](https://github.com/user-attachments/assets/88307d91-a69d-4892-b878-da564d55e773)
